docs/FunctionBlocks/FB_MQTT_DEVICE.md

```diff
@@ -0,0 +1,44 @@
+## FB_MQTT_DEVICE
```

Comment:
Q1. You are probably right. It is nicer to move it. There will be an other of execution of the methods. First initmqtt, than initdmx, and then mqttdiscover.  Otherwise you don't get this 'dmx metadata'

Q2. The good thing about the Stefans json libary, it kicks out the keys that are null. So all possible keys can be defined in MQTT_DISCOVERY but every FB needs to define a subset. a switch needs less than a dimmer.